app/models/setting.py

```diff
@@ -222,6 +230,12 @@ def __init__(self,
                  omise_live_secret=None,
                  alipay_publishable_key=None,
                  alipay_secret_key=None,
+                 is_paytm_activated=None,
```

Comment:
this should be `is_paytm_activated=False`

Reply:
Taken care of in the recent iteration.